receiver/zipkinreceiver/trace_receiver.go

```diff
@@ -175,10 +174,10 @@ func (zr *ZipkinReceiver) v2ToTraceSpans(blob []byte, hdr http.Header) (reqs []c
 	}
 
 	if err != nil {
-		return nil, err
+		return pdata.Traces{}, err
 	}
 
-	return zipkin.V2BatchToOCProto(zipkinSpans)
```

Comment:
Can the old method `V2BatchToOCProto` be removed?

Reply:
Deleted